js/Makefile

```diff
@@ -40,9 +40,21 @@ languages:
 engine:
 	cd engine; $(MAKE)
 
+.PHONY: pro-setup
+pro-setup:
```

Comment:
Why can't this be done in a semgrep-pro/js/Makefile instead?
Would be good to document somewhere why we do it that way, what were the pro/cons (example of cons was having to duplicate lots of code such as XX and YY)

Reply:
We could move it there, this just seemed simplest to me to reuse things (instructions stay exactly the same, just add an environment variable). I'll add a comment explaining this

Reply:
I'd really prefer it if we kept pro-specific stuff in the pro. It feels wrong to have proprietary stuff in the OSS makefile. if it's essential that keep these changes in the OSS repo, can you help me better understand why?

Reply:
It's not essential, just a choice I made, I can move it to OSS